src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -188,7 +188,9 @@ internal static class IDEDiagnosticIds
         // public const string ErrorReadingRulesetId = "IDE1004";
         public const string InvokeDelegateWithConditionalAccessId = "IDE1005";
         public const string NamingRuleId = "IDE1006";
-        public const string UnboundIdentifierId = "IDE1007";
+
+        // Deprecated. This was a workaround for missing compiler binding errors that are now fixed in the compiler.
+        //public const string UnboundIdentifierId = "IDE1007";
```

Comment:
>//public const string UnboundIdentifierId = "IDE1007"; [](http://example.com/codeflow?start=8&length=54)

Are you absolutely certain that incomplete member is the only scenario that can trigger this diagnostics?

Reply:
I think yes.

https://github.com/dotnet/roslyn/blob/26ac760823603fb0be65db030ef7d17bcda356db/src/Features/Core/Portable/Diagnostics/Analyzers/UnboundIdentifiersDiagnosticAnalyzerBase.cs#L50

https://github.com/dotnet/roslyn/blob/26ac760823603fb0be65db030ef7d17bcda356db/src/Features/VisualBasic/Portable/Diagnostics/Analyzers/VisualBasicUnboundIdentifiersDiagnosticAnalyzer.vb#L17-L19

https://github.com/dotnet/roslyn/blob/26ac760823603fb0be65db030ef7d17bcda356db/src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpUnboundIdentifiersDiagnosticAnalyzer.cs#L21-L23